src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx

```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
 tags:
   - Integrations
   - Amazon integrations
-  - AWS integrations list
+  - Lambda Extension
 ---
+**Work in progress**
 
-The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and setting up your New Relic license key as an environment variable.
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and get insights from your data instently on New Relic One platform.
 
 ## Installation
 
-
 1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
 2. Select **Layers** > **Add Layer**.
 3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
-4. Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
+4a. If you can't find the extension as described in step 4, use the following pattern to get ARN <var>arn:aws:lambda:{your-region}:533243300146:layer:AwsLambdaExtension:{version}</var> The easiest way to get the most recent version would be to start from 1 and save, then push "Edit" button and select the highest version available for the chosen extension.  
+5. Go to **Configuration**, select **Environment variables** and add **Key: NEW_RELIC_LICENSE_KEY  Value: <var>YOUR_LICENSE_KEY</var>**. **Key: NEW_RELIC_ACCOUNT_ID** is an optional parameter.
```

Comment:
could we add: "It has to be set to your Account ID on New Relic platform, if you want events data" after "NEW_RELIC_ACCOUNT_ID** is an optional parameter"?